pkg/plugin/v3/scaffolds/internal/templates/controllers/controller_suitetest.go

```diff
@@ -159,7 +159,8 @@ var _ = BeforeSuite(func() {
 
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join({{ .CRDDirectoryRelativePath }}, "config", "crd", "bases")},
+		CRDDirectoryPaths:     []string{filepath.Join({{ .CRDDirectoryRelativePath }}, "config", "crd", "bases")},
+		ErrorIfCRDPathMissing: {{ .WireResource }},
```

Comment:
We need to do the same for webhooks_suitetest as well.

Reply:
This is more difficult on webhook_suitetest. There's no doResource/WireResource boolean that can be used, the closest point i can see where a doResource is actually used is harcoded to false: https://github.com/kubernetes-sigs/kubebuilder/blob/21de76043e6e14fd775729452c3c2a1a3432ad8e/pkg/plugins/golang/v3/webhook.go#L131 so i can just hardcode it to false as well or ignore it